charts/tempo-distributed/templates/gateway/deployment-gateway.yaml

```
@@ -101,6 +101,9 @@ spec:
           securityContext:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+        {{- with .Values.gateway.extraContainers }}
+        {{- toYaml .Values.gateway.extraContainers | nindent 8 }}
```

```suggestion
        {{- toYaml . | nindent 8 }}
```
Inside the `{{- with .Values.gateway.extraContainers }}` block, the scope has already been changed to `.Values.gateway.extraContainers` context, therefore to render `extraContainers`, just use `.`  inside the block.

Otherwise it throws this error, during testing:

`Error: template: tempo-distributed/templates/gateway/deployment-gateway.yaml:105:26: executing "tempo-distributed/templates/gateway/deployment-gateway.yaml" at <.Values.gateway.extraContainers>: can't evaluate field Values in type []interface {}`

---

sorry, was actually surprised the tests passed